charts/cheetah-redpanda-console/templates/deployment.yaml

```diff
@@ -0,0 +1,348 @@
+            {{- end }}
+            {{- if .Values.secret.enterprise.license }}
+            - name: LICENSE
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "console.fullname" . }}
+                  key: enterprise-license
+            {{- end }}
+            {{- if .Values.secret.redpanda.adminApi.password }}
+            - name: REDPANDA_ADMINAPI_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "console.fullname" . }}
+                  key: redpanda-admin-api-password
+            {{- end }}
+            {{- if .Values.secret.redpanda.adminApi.tlsCa }}
+            - name: REDPANDA_ADMINAPI_TLS_CAFILEPATH
+              value: "/etc/console/secrets/redpanda-admin-api-tls-ca"
+            {{- end }}
+            {{- if .Values.secret.redpanda.adminApi.tlsKey }}
+            - name: REDPANDA_ADMINAPI_TLS_KEYFILEPATH
+              value: "/etc/console/secrets/redpanda-admin-api-tls-key"
+            {{- end }}
+            {{- if .Values.secret.redpanda.adminApi.tlsCert }}
+            - name: REDPANDA_ADMINAPI_TLS_CERTFILEPATH
+              value: "/etc/console/secrets/redpanda-admin-api-tls-cert"
+            {{- end }}
+          {{- else }}
+            {{- if dig "licenseSecretRef" "name" "" .Values.enterprise }}
+            - name: LICENSE
+              valueFrom:
+                secretKeyRef:
+                  name: {{ dig "licenseSecretRef" "name" "" .Values.enterprise }}
+                  key: {{ dig "licenseSecretRef" "key" "enterprise-license" .Values.enterprise }}
+            {{- end }}
+          {{- end }}
+          {{- with .Values.extraEnvFrom }}
+          envFrom:
+              {{- toYaml . | nindent 12 }}
+          {{- end }}
+        - name: oauth2-proxy
+          image: quay.io/oauth2-proxy/oauth2-proxy:v7.5.1
+          imagePullPolicy: IfNotPresent
+          args:
+            - --oidc-issuer-url={{.Values.oauth2Proxy.issuerUrl}}
+            - --upstream=http://127.0.0.1:8080
+            - --reverse-proxy=true
+            - --email-domain={{.Values.oauth2Proxy.emailDomain}}
+            - --cookie-secure=true
+            - --cookie-name={{.Values.oauth2Proxy.cookieName}}
+            - --provider=oidc
+            - --http-address=0.0.0.0:4180
+            - --exclude-logging-path="/ping"
+          readinessProbe:
+            httpGet:
+              path: /ping
+              port: http-proxy
+            initialDelaySeconds: 5
+            periodSeconds: 5
+          livenessProbe:
+            httpGet:
+              path: /ping
+              port: http-proxy
+            initialDelaySeconds: 5
+            periodSeconds: 5
+          securityContext:
+            runAsNonRoot: true
+            allowPrivilegeEscalation: false
+            readOnlyRootFilesystem: true
+            capabilities:
+              drop:
+                - ALL
+          ports:
+            - containerPort: 4180
```

Comment:
Should preferably also be taken from values, and should just match https://github.com/trifork/cheetah-charts/pull/216/files#diff-7f4165ef21ea4ac14fc7bc4e70899250ef7adeddae686abe79631a21aaa40e8eR29